source/agora/consensus/protocol/Nominator.d

```diff
@@ -945,7 +945,7 @@ extern(D):
             }
             const block = makeNewBlock(last_block,
                 externalized_tx_set, data.time_offset, random_seed,
-                validators.length,
+                validators.length - data.missing_validators.length,
```

Comment:
I think this is wrong as the number of validators set in the BitMask in the block header is not excluding the missing validators.
![Sprint #46 - Issues 1669 + 2229 002](https://user-images.githubusercontent.com/11815575/127243433-35c7f1c2-681d-481e-a635-0d7259088191.jpeg)

Reply:
Thanks for the help, indeed that was it.